gramps/plugins/drawreport/descendtree.py

```diff
@@ -71,7 +72,11 @@
 _MARR = (_("m.", "marriage abbreviation"),)
 
 _RPT_NAME = "descend_chart"
-
+_BOXCOLOR_DEFAULT = (255,255,255) # default dark
```

Comment:
Can you please use the users default colors already defined in the Colors preference tab see

https://gramps-project.org/wiki/index.php/Gramps_5.2_Wiki_Manual_-_Settings#Colors